src/Microsoft.TestPlatform.Extensions.TrxLogger/ObjectModel/TestMethod.cs

```diff
@@ -24,7 +24,8 @@ public TestMethod(string name, string className)
         {
             Debug.Assert(!string.IsNullOrEmpty(name), "name is null");
             Debug.Assert(!string.IsNullOrEmpty(className), "className is null");
-            this.name = name;
+            this.name = name.StartsWith(className) ? name.Remove(0, $"{className}.".Length) : name;
```

Comment:
Should we do this in the Converter itself ?

Reply:
Yes. done